src/components/dialogs/line-attach-to-voltage-level-dialog.js

```diff
@@ -499,39 +500,38 @@ const LineAttachToVoltageLevelDialog = ({
                     </Grid>
                     <GridSection title="VoltageLevel" />
                     <Grid container spacing={2}>
-                        {gridItem(voltageLevelIdField, 5)}
+                        {gridItem(voltageLevelIdField, 6)}
+                        {gridItem(bbsOrNodeIdField, 6)}
                         {gridItem(
                             <Button
                                 onClick={openVoltageLevelDialog}
                                 startIcon={
                                     newVoltageLevel ? <EditIcon /> : <AddIcon />
                                 }
-                                variant="outlined"
                             >
                                 <Typography align="left">
                                     <FormattedMessage id="NewVoltageLevel" />
                                 </Typography>
                             </Button>,
-                            2
+                            6
                         )}
-                        {gridItem(bbsOrNodeIdField, 5)}
                     </Grid>
                     <GridSection title="AttachedLine" />
                     <Grid container spacing={2}>
-                        {gridItem(lineToIdField, 4)}
+                        {gridItem(lineToIdField, 6)}
+                        <Box width="100%" />
```

Comment:
What is this Box ?

Reply:
It's the way to go if you want a line break in a grid